docs/marketplace/dev/architecture-overview/202108.0/marketplace-architecture-overview-application-composition.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Marketplace Application Composition
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace looks like in general. After reading this odc you will now: 
+
+- how the Spryker Marketplace is composed on application level
```

Comment:
- what Marketplace setups are available (B2C/B2B Marketplace, Standalone Marketplace (other shop sys serve as an order channel, (maybe?) Unitifed Commerce setup (where merchants are filials or offline stores).

Reply:
What do you mean by Standalone Marketplace? - Standalone MerchantPortal?
b2c/b2b will looks the same.
unified commerce also